### PR TITLE
CP-JKU: Add model 'mn40_as' from paper 'https://arxiv.org/abs/2211.04772'

### PR DESCRIPTION
**Pull request by fschmid56**
CP-JKU

Dear HEAR committee,

I am requesting a review of my submission to the HEAR Benchmark leaderboard:

**Team Name:** CP-JKU

**Model Name:** CP-JKU mn40_as (e1+l+all_se)

**Author(s):** Florian Schmid, Khaled Koutini, Gerhard Widmer

**Model URL/GitHub:** https://github.com/fschmid56/EfficientAT_HEAR

**Paper Link:** https://arxiv.org/pdf/2211.04772.pdf

**Model Description:**

The architecture is based on the efficient MobileNetV3 CNN-architecture. Models are trained on AudioSet using Knowledge Distillation from Patchout faSt Spectrogram Transformers (PaSST).

**Pre-training Data:**
* [ ] speech
* [X] broad-domain
* [ ] music

**Number of Model Trainable Parameters:**

68.43 Million parameters

**Scene Embedding Dimensions:**

8423

**Timestamp Embedding Dimensions:**

8423

**Were you able to evaluate your model on all 19 tasks?:**
* [X] yes
* [ ] no

**If you answered no to the last question, list which tasks you were unable to complete and why:**

---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Committee acknowledgment
* [ ] Committee decision [accept/reject]

### CODE REVIEW

**Comment by jorshi** [comment]
Dear @fschmid56, thank you for your submission to the HEAR benchmark leaderboard. We will review this submission in the coming days.

File: docs/leaderboard.csv [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+Stellenbosch LSL DBERT,https://github.com/hearbenchmark/hear2021-submitted-models,0.6972415149,0.9191489458,0.5215128779,0.2462006211,0.5315000057,0.2634900212,0.6740000069,0.968589747,0.8095238124,0.5842657328,0.0001634800719,0.8351714611,0.6848203301,0.7374384403,0.523999989,0.565644145,0.6304703355,0.1611349632,0.2458524972
+wav2vec2,https://github.com/hearbenchmark/hear2021-submitted-models,,0.9067376018,0.6562244773,0.6629955769,0.5610000014,0.341744781,0.7800000012,0.9461538494,0.8482142942,0.6921328664,0.03289615512,0.9432430506,0.8282943606,0.6529556513,0.40200001,0.8382412791,0.8785275817,0.08005668471,0.4928152204
+CP-JKU mn40_as (e1+l+all_se),https://github.com/fschmid56/EfficientAT_HEAR,0.5026644483,0.9533687949,0.6399549484,0.8130252361,0.961499989,0.6312075533,0.9079999983,0.9839743614,0.9107142857,0.7253496408,0.0186244165,0.9713343859,0.9046840429,0.6354680061,0.3039999902,0.7766871452,0.8472392559,0.2019387526,0.3179222822
```
Comment (jorshi) [inline docs/leaderboard.csv:31]: Please remove the newline at the end of this csv file as it adds a blank row to the table.

**Review by jorshi — CHANGES_REQUESTED** [review]
Thank you for submitting such a thorough PR complete with model details and links to the model code and instructions for reproducing. I've requested one minor correction to the csv file. Once that is complete we will publish these results and then merge with the second submission #8.

File: docs/leaderboard.csv (hunk 1 above)
Comment (fschmid56) [reply docs/leaderboard.csv]: Dear @jorshi, thanks for reviewing! I've struggled a bit with the GitHub online editor which insists on CSV files ending with a linebreak but it should be done now for both branches.

**Pull request merged**